tests/test_loader.py

```diff
@@ -165,6 +166,24 @@ def test_yaml_load_exploit(self):
             # If the command in exploit.yaml is echoed it will return 0
             self.assertNotEqual(config, 0)
 
+    def test_env_var_regex(self):
+        test_data = [
+            {"env_var": "$OPS_DROID", "match": True},
+            {"env_var": "$OPS-DROID", "match": True},
+            {"env_var": "${OPS_DROID}", "match": True},
+            {"env_var": '"$OPSDROID_SLACK_TOKEN"', "match": True},
+            {"env_var": "$INVALID!_CHARACTERS@", "match": False},
+            {"env_var": "OPS_DROID", "match": False},
+            {"env_var": "$OPS_DROID23", "match": False},
+            {"env_var": "$UPPER-AND-lower", "match": False},
+            {"env_var": '"MISSING_PREFIX"', "match": False},
+        ]
+        for d in test_data:
+            # Tests opsdroid constant ENV_VAR_REGEX for both valid and invalid environment variables.
+            self.assertRegex(d["env_var"], ENV_VAR_REGEX) if d[
+                "match"
+            ] else self.assertNotRegex(d["env_var"], ENV_VAR_REGEX)
+
```

Comment:
This is a nice implementation but this is a little hard to read. I don't think black has helped you here either.

Would you mind changing it to 

```python
if d["match"]:
    self.assertRegex(d["env_var"], ENV_VAR_REGEX)
else:
    self.assertNotRegex(d["env_var"], ENV_VAR_REGEX) 
```

Reply:
No problem!